### ISSUE TITLE
Upload event: Several issues (Cancel-Button missing, wrong/misleading wording, default start time not shown)

### ISSUE THREAD

**Issue by dagraf** [opencast-ilias/OpenCast#55]
**Describe the bug**
In the form to upload an event there are several bugs (see below).

**To Reproduce**
1. Go to Opencast series where you can upload an event
2. Click on 'Upload Event'

**Expected behavior**
1. There should be a button 'Cancel' next to the button 'Save'.
2. The wording 'Select files' is misleading it assumes users can upload multiple videos/audios. Please replace by 'Select file'.
3. Below the option to select files the 'Supported filetypes' should be indicated.
4. The actual date and time should be prefilled by default in the box 'Start'.
5. The title of the section should be renamed to 'Settings'

**Screenshots**
v4.0.1:
<img width="854" alt="Bildschirmfoto 2022-06-20 um 22 24 42" src="https://user-images.githubusercontent.com/15246287/174674106-a2d0dbda-5dbd-4994-9ffa-761150b45f7f.png">

**Comment by reiferschris** [comment]
Could you specify which section you want to rename to settings? Do you mean the metadata section? If  you do, I wouldn't agree because this section only contains metadata which we grouped together to avoid confusion.

**Comment by dagraf** [comment]
No, I mean the section called "Workflow Parameter". In my opinion that's a very techy expression and could confuse some users. Therefore I would propose to change it to "Settings". What do you think?

**Comment by reiferschris** [comment]
That sounds totally reasonable. I don't know why that didn't cross my mind. We could add something along the lines of `processing settings` or `Verarbeitungsschritte/Verarbeitungseinstellungen` to make it a bit clearer what is happening. This could avoid confusion for customer support later because `settings` could also apply to course settings or object settings.

**Comment by dagraf** [comment]
I'm fine with "Processing settings" and "Verarbeitungseinstellungen". Or just "Processing" and "Verarbeitung"?

**Comment by reiferschris** [comment]
Unfortunately changing the name of the section `Workflow Parameter` is not as easy as I thought @dagraf since it is hard-coded here https://github.com/opencast-ilias/OpenCast/blame/main/src/Model/WorkflowParameter/Series/SeriesWorkflowParameterRepository.php#L326
Sorry!
I could try to fix this but it would take me a while..

**Comment by dagraf** [comment]
@reiferschris Thanks for having had a look at it. We will therefore change it after the major release.

**Comment by rfcmaXi** [comment]
Hello everyone,
We are now slowly moving to the new community plugin and have come across the same "bug" that david mentioned:
Why is the Start field not prefilled with the current date? As I understand the metadata, there is currently no placeholder for this?
Thank you!

**Comment by dagraf** [comment]
@chfsx: Can the current date and time somehow be configured? I tried [NOW], without success.
<img width="824" alt="Bildschirmfoto 2024-02-12 um 15 05 53" src="https://github.com/opencast-ilias/OpenCast/assets/15246287/7073e90a-470b-4147-81a8-9aefe4d10b87">

**Comment by chfsx** [comment]
The ticket contains several items:

*wording 'Select files'*
This is predefined by the UI component, unfortunately we cannot influence this in the plugin

* prefill NOW in startDate
from my point of view, this was never implemented, i think it needs an extension of the available placeholders. from my point of view, a change-request

*'Workflow parameters' to 'Processing'*
done

**Comment by dagraf** [comment]
Wording 'select files': Ok. We can live with that wording, I guess.

Prefill startDate: In the old plugin version (before the implementation of configurable metadata and the refactoring done by fluxlab), the prefill of the startDate with now was implemented. I will open a change-request in the couple of next days/weeks. I will then close this ticket and refer to the change-request.

**Comment by dagraf** [comment]
Prefill startDate: The actual behavior seems sufficient to me. If I do not indicate a date, the current time and date is set. And if I hit on the date picker the same happens.

Therefore, I close this issue.
